doc/classes/TileMap.xml

```diff
@@ -173,6 +173,31 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="set_cell_region">
+			<return type="void">
+			</return>
+			<argument index="0" name="start" type="Vector2" default="Vector2( 0, 0 )">
+			</argument>
+			<argument index="1" name="end" type="Vector2" default="Vector2( 0, 0 )">
+			</argument>
+			<argument index="2" name="tile" type="int" default="-1">
+			</argument>
+			<argument index="3" name="flip_x" type="bool" default="false">
+			</argument>
+			<argument index="4" name="flip_y" type="bool" default="false">
+			</argument>
+			<argument index="5" name="transpose" type="bool" default="false">
+			</argument>
+			<argument index="6" name="autotile_coord" type="Vector2" default="Vector2( 0, 0 )">
+			</argument>
+			<description>
+				Sets the tiles in the given region.
+				An index of [code]-1[/code] clears the region.
+				Optionally, the tile can also be flipped, transposed, or given autotile coordinates. The autotile coordinate refers to the column and row of the subtile.
+				[b]Note:[/b] Data such as navigation polygons and collision shapes are not immediately updated for performance reasons.
```

Comment:
Do you know exactly when they update? If on the next physics step for instance, it'd be useful to mention it. The more precise the reference, the better.

Reply:
It's the same as the method set_cell but for a region. This method use a call_defered for the update_dirty_quadrants. I think it is, but I'm not pretty sure that this update is in the next physics step.

Reply:
When something is deferred, it's always called on the idle frame. Not sure if it's even worth mentioning.